main.js

```diff
@@ -0,0 +1,43 @@
+var chkBook = document.querySelector('.expenses');
+
+var addstuffList = function(event) {
+  var itemsBought = document.getElementById('expenses');
+  var costSpent = document.getElementById('amount');
```

Comment:
You can keep these outside of your function so you only have to query the DOM once.